runtime/near-vm-logic/src/config.rs

```diff
@@ -134,8 +134,9 @@ impl Default for VMLimitConfig {
             // Total logs size is 16Kib
             max_total_log_length: 16 * 1024,
 
-            // Fills 10 blocks. It defines how long a single receipt might live.
-            max_total_prepaid_gas: 10 * 10u64.pow(15),
+            // Updating the maximum prepaid gas to limit the maximum depth of a transaction to 64
+            // blocks.
+            max_total_prepaid_gas: 300 * 10u64.pow(12),
```

Comment:
How 300 is calculated?

Reply:
It's based on `63 * min_receipt_with_function_call_gas()` which is `305918046000000`.